FWCore/Framework/interface/ESProducts.h

```diff
@@ -28,114 +28,113 @@
 
 // forward declarations
 namespace edm {
```

Comment:
Are the only changes the spacing and the collapsed namespace? If so, please revert.

Reply:
Yes, will revert.